trunk/SUAVE/Input_Output/OpenVSP/vsp_write.py

```diff
@@ -92,6 +93,7 @@ def write(vehicle,tag,fuel_tank_set_ind=3):
     """    
     
     # Reset OpenVSP to avoid including a previous vehicle
+    print('Reseting OpenVSP Model in Memory')
```

Comment:
put a flag if verbose to add printing.

I've made 400 VSP vehicles at once. I don't want prints.